addons/smtp-mail.sh

```diff
@@ -205,7 +205,6 @@ fi
 chmod 600 /etc/msmtprc
 
 # Create logs
-# TODO: not working due to permissions error
```

Comment:
Is this still true?

Reply:
When you tested it and it works as expected then it's true. :)

Reply:
still permission error, but I rewrite it manually to the logfile. So this is kind of resolved.